services/app-api/handlers/banners/create.test.ts

```diff
@@ -21,9 +21,18 @@ const testEvent: APIGatewayProxyEvent = {
   pathParameters: { bannerId: "testKey" },
 };
 
+const consoleSpy: {
+  debug: jest.SpyInstance<void>;
+  error: jest.SpyInstance<void>;
+} = {
+  debug: jest.spyOn(console, "debug").mockImplementation(),
+  error: jest.spyOn(console, "error").mockImplementation(),
+};
+
```

Comment:
is there any way we could make this shared in the setupJest file?

Reply:
Possibly. I looked into it a little bit, but didn't find a ready answer. Open to refactoring this in another cleanup ticket.

Reply:
One reason I wouldn't want to set this up globally is I would want to catch console messages we aren't expecting.